### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by Vitalii-Kazymyrovych**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+   - MYSQL_USER=**YOUR MYSQL USERNAME**
+   - MYSQL_PASSWORD=**YOUR MYSQL USER PASSWORD**
+   - MYSQL_ROOT_PASSWORD=**YOUR MYSQL ROOT PASSWORD**
+   - MYSQL_DATABASE=**YOUR MYSQL DB NAME**
+   - MYSQL_LOCAL_PORT=3307
+   - MYSQL_DOCKER_PORT=3306
+   - SPRING_LOCAL_PORT=8088
+   - SPRING_DOCKER_PORT=8080
+   - DEBUG_PORT=5005
```
Comment (koval-kn) [inline README.md]: create .env.sample for that

File: .env [hunk 2]
```diff
@@ -1,6 +1,6 @@
 MYSQL_USER=vitalii
-MYSQL_PASSWORD=1469
-MYSQL_ROOT_PASSWORD=1469
+MYSQL_PASSWORD=14691769
+MYSQL_ROOT_PASSWORD=14691769
 MYSQL_DATABASE=bookstore
 MYSQL_LOCAL_PORT=3307
 MYSQL_DOCKER_PORT=3306
```
Comment (koval-kn) [inline .env]: this file should not be pushed to any open sources. it keeps your secrets. create .env.sample with mock data instead

**Review by koval-kn — CHANGES_REQUESTED** [review]

File: .env [hunk 3]
```diff
@@ -1,10 +1,10 @@
-MYSQL_USER=vitalii
```
Comment (Elena-Bruyako) [inline .env:1]: rename file to .env.sample according to the previous mentor comment

File: README.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
+# Bookstore Spring Boot Project
```
Comment (Elena-Bruyako) [inline README.md:1]: Also, you can add Database structure (Models and relations)

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
almost perfect!

**Review by okuzan — APPROVED** [review]

**Pull request merged**